.github/STYLEGUIDE.md

```diff
@@ -743,6 +743,37 @@ Each guide is broken into a few sections. Sections contain a list of guidelines.
   ```
   </details>
 
+* **PREFER attributes on the same line as what they are attributing.**
+
+  <details>
+
+  #### Why?
+
+    Because we naturally read lines of code as a consistent line, introducing whitespace makes it that much harder to understand the full context of the line you're trying to understand. This is the same reasoning we put behind preferring a return on the same line as the final parameter in a function declaration.
```

Comment:
"applied to" is fine. I am also not sure I like yours better, fun fact...

Reply:
```suggestion
    Because we naturally read lines of code as a consistent line, introducing whitespace makes it that much harder to understand the full context of the line you're trying to understand. This is the same reasoning used for preferring a return on the same line as the final parameter in a function declaration.
```

Not sure I like this one any better than "applied to"

Reply:
If we keep this line for splitting into 2 rules, I think this one reads better.

Reply:
I like `used for` here